build_env.rst

```diff
@@ -261,6 +261,15 @@ meaning that no intermediate decrypted data is ever present on disk. See
 Environment Variables
 *********************
 
+You can set environment variables on the host to influence the behaviour of a
+build. Note that environment variables are not passed into the build itself, and
+cannot be accessed in the ``%post`` section of a definition file. To pass values
+into a build, use the :ref:`templating / build-args support <sec:templating>`
+introduced in {Project} 4.0.
```

Comment:
Remove `introduced in {Project} 4.0` here and also below.

Reply:
thanks! good catch.